src/Calypso-SystemTools-Core/SycRemoveMethodStrategy.extension.st

```diff
@@ -13,6 +13,18 @@ SycRemoveMethodStrategy class >> createAllForBrowser: aBrowser [
 	^result
 ]
 
+{ #category : #'*Calypso-SystemTools-Core' }
+SycRemoveMethodStrategy class >> createForBrowser: aBrowser methods: aCollection [
```

Comment:
I would name this method a bit different:
```Smalltalk
SycRemoveMethodStrategy createAllForMethods: methods browser: browser
```
It would follow the name of method #createAllForBrowser:.